blog-service/2024-05-31-dashboards.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Multiple Scheduled Reports (Dashboards)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - search
+  - scheduled_reports
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We are excited to introduce the **Multiple Scheduled Reports** feature to Sumo Logic dashboards. This powerful new capability allows you to configure multiple reports against a dashboard, providing enhanced flexibility and control over your reporting needs. Key features include:
+
+- **Multiple Configurations**. Set up several reports from the same dashboard, each with its unique settings.
+- **Custom Variable Values**. Assign different variable values to each report, tailoring the data to meet specific reporting needs.
+- **Custom Time Ranges**. Define distinct time ranges for each report, ensuring you capture the data most relevant to your analysis.
+- **Run Frequencies**. Schedule reports to run at varying frequencies, from daily snapshots to monthly summaries.
+- **Email Addresses**. Specify different email addresses for each report, ensuring the right information reaches the right stakeholders.
```

Comment:
Suggestion:
different email addresses for each report-> one or more email addresses for each report